src/styleguide/components/s-theme-selector.vue

```diff
@@ -110,7 +110,18 @@
        * @param {Event} event - The related DOM event.
        */
       onChange(event) {
-        this.$store.commit('session/setTheme', event.target.value);
+        const themeId = event.target.value;
+        const urlParam = new URLSearchParams(window.location.search);
+
+        this.$store.commit('session/setTheme', themeId);
+
+        if (urlParam.has('theme')) {
+          urlParam.delete('theme');
+        }
+
+        urlParam.append('theme', themeId);
```

Comment:
todo: you can just use `urlParam.set()` instead of this three lines. No need to delete the old value first.

@see https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/set